frontend/components/auth/google.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
```

Comment:
FYI I tend to import { Component } from 'react' and just extend Component but either is fine